linux/linux-assessment.md

```diff
@@ -799,6 +799,14 @@ if [[ $FILE == $GLOB ]] ;then
 - [ ] rename
 - [ ] ren
 
+
+#### Q96. Which of these do NOT constitute as a default part of the email system of Red Hat Linux?
```

Comment:
How is this related to HTML? Looks like it is a good question, but not really fit for HTML quiz.